app/models/experiment.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import Experiment from 'exp-models/models/experiment';
+import SessionModel from '../models/session';
+
+
+export default Experiment.extend({
+  _registerSessionModels() {
+      // Register the session model defined in ISP
```

Comment:
This is an extreme hack. Although we recognize the present need, we should add a source code comment explaining the context for this being here, and possibly tagging this for followup at a later date. 

Otherwise a later maintainer would likely see this as redundant and remove it, triggering a regression.